src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -82,10 +89,31 @@ public DebugService(PowerShellContext powerShellContext)
 
             if (breakpoints.Length > 0)
             {
+                // Make sure we're using the remote script path
+                string scriptPath = scriptFile.FilePath;
+                if (this.powerShellContext.CurrentRunspace.Locale == RunspaceLocale.Remote)
```

Comment:
Locale might be confused with i18n "locale".  Or perhaps you don't have control over that name.